### PR TITLE
Feat: completed likedvideo and watch latercontext

### PR DESCRIPTION
**Pull request by tewarimohit**

### CODE REVIEW

File: src/Components/LikedVideoCard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "../Styles/Videos.css";
+import { ThumbUpAltRounded } from "@material-ui/icons";
```
Comment (Mohitkumar33) [inline src/Components/LikedVideoCard.js:3]: You can put these in one line

File: src/Components/LikedVideoCard.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { WatchLaterRounded } from "@material-ui/icons";
+import { useLikedVideos } from "../Context/LikedVideosContext";
+
+const LikedVideoCard = () => {
+	const { likedVideos, setLikedVideos } = useLikedVideos();
+	console.log(likedVideos);
```
Comment (Mohitkumar33) [inline src/Components/LikedVideoCard.js]: remove console.log

File: src/Components/LoginArea.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+	const userInputHandler = event => {
+		setUserLogin(data => ({
+			...data,
+			[event.target.name]: event.target.value,
+		}));
+	};
+
+	const userLoginHandler = async (email, password) => {
+		console.log(email, password);
```
Comment (Mohitkumar33) [inline src/Components/LoginArea.js]: remove console

File: src/Components/LoginArea.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+			authDispatch({
+				type: "USER-LOADING-SUCCESSFUL",
+				payload: response.data.foundUser,
+			});
+
+			localStorage.setItem("token", response.data.encodedToken);
+
+			navigate("/");
+		} catch (error) {
+			console.log(error.response.data);
```
Comment (Mohitkumar33) [inline src/Components/LoginArea.js:39]: you can console.error instead of console.log

File: src/Components/LoginArea.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+				>
+					Remember me
+				</label>
+				<Link to="" className="margin-auto heading-color ">
+					Forgot your Password ?
+				</Link>
+			</div>
+
+			<button
+				class="primary-btn normal-btn btn-width"
```
Comment (Mohitkumar33) [inline src/Components/LoginArea.js:94]: convert class to className

File: src/Components/VideosCard.js [hunk 6]
```diff
@@ -2,31 +2,69 @@ import React from "react";
...
+import { useNavigate } from "react-router-dom";
+import { useWatchLater } from "../Context/WatchLaterContext";
+import { addToWatchLater } from "../Utility.js/WatchLaterUtil";
 
-const VideoCard = ({ image, title, uploaded, views, description }) => {
+const VideoCard = ({ video }) => {
+	const navigate = useNavigate();
+	const { likedVideos, setLikedVideos } = useLikedVideos();
+	const { watchLater, setWatchLater } = useWatchLater();
+	console.log(typeof (navigate) );
```
Comment (Mohitkumar33) [inline src/Components/VideosCard.js]: you can remove console.log

File: src/Components/VideosCard.js [hunk 7]
```diff
@@ -2,31 +2,69 @@ import React from "react";
...
-					<button className="icons">
-						<ThumbUpAltRounded  />
+					<p>1000 likes</p>
+					<button
+						className="icons"
+						title="Like"
+						onClick={() => {
+							if (getToken()) {
+								if (likedVideos.find(item => item._id === video._id)) {
+									console.log("Item is already there.");
```
Comment (Mohitkumar33) [inline src/Components/VideosCard.js:36]: you can remove console

File: src/Components/VideosCard.js [hunk 8]
```diff
@@ -2,31 +2,69 @@ import React from "react";
...
 
-					<button className="icons">
-					<WatchLaterRounded />
+					<button
+						className="icons"
+						title="Watch Later"
+						onClick={() => {
+							if (getToken()) {
+								if (watchLater.find(item => item._id === video._id)) {
+									console.log("Item is already there.");
```
Comment (Mohitkumar33) [inline src/Components/VideosCard.js:54]: here also console

**Review by Mohitkumar33 — APPROVED** [review]
code is clean

File: src/Components/Header.js [hunk 9]
```diff
@@ -1,26 +1,61 @@
...
+					<div className="std-font-two hd-m2-font title ">woWWatch</div>
+				</Link>
 			</div>
 			<div className="header-search">
 				<input type="text" placeholder="Search" />
 				<SearchRounded className="search-btn" />
 				<MicNoneOutlined className="search-btn" />
 			</div>
 			<div className="header-profile">
+			{authState.isAuth ? (
```
Comment (deepak29-git) [inline src/Components/Header.js:46]: suggestion: you can destructure it from authstate and use only isAuth

File: src/Context/LikedVideosContext.js [hunk 10]
```diff
@@ -1,10 +1,33 @@
...
+		(async () => {
+			try {
+				const response = await axios.get("/api/user/likes", {
+					headers: {
+						authorization: getToken(),
+					},
+				});
+
+				setLikedVideos(response.data.likes);
+				console.log(response.data);
```
Comment (deepak29-git) [inline src/Context/LikedVideosContext.js:24]: remove console

File: src/Styles/Header.css [hunk 11]
```diff
@@ -36,3 +36,15 @@
 .header-icons {
 	margin-right: 10px;
 }
+
+.login-color {
+	background-color: #f5fffa;
```
Comment (deepak29-git) [inline src/Styles/Header.css:41]: use color variable

File: src/Styles/Login.css [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+}
+.margin-bottom-s {
+	margin-bottom: 0.2rem;
+}
+.margin-bottom-m {
+	margin-bottom: 0.5rem;
+}
+
+.normal-btn {
+	background-color: #f5fffa;
```
Comment (deepak29-git) [inline src/Styles/Login.css:92]: use color variable

**Review by deepak29-git — APPROVED** [review]

**Pull request merged**